src/dev/precommit_hook/casing_check_config.js

```diff
@@ -40,6 +40,7 @@ export const IGNORE_FILE_GLOBS = [
   'vars/*',
   '.ci/pipeline-library/**/*',
   'packages/kbn-test/jest-preset.js',
+  'packages/kbn-test/jest_integration/jest-preset.js',
```

Comment:
Why can't we use `jest_preset`?

Reply:
@mshustov jest doesn't allow us to configurate wether to make or not the workers to follow symlinks (except in places where we can use a custom resolver, so mostly within test code). Therefore we discover that if we keep the reporters config hold inside the @kbn/test package other than overridden outside, we are able to keep the preserve symlink option set in the parent process and correctly discover the output folder for the reports on the CI

Reply:
I think he's asking specifically about the use of kebab-case instead of camel_case

Reply:
🤦 thanks @spalger , reading it a second time make that clear. Yeah it is a jest limitation, it will always look for a `jest-preset.*` file. More info at https://jestjs.io/docs/configuration#preset-string